doc/JSON_FLAGS.md

```diff
@@ -1311,6 +1213,7 @@ Melee flags are fully compatible with tool flags, and vice versa.
 - ```RECHARGE``` Gain charges when placed in a cargo area with a recharge station.
 - ```SAFECRACK``` This item can be used to unlock safes.
 - ```USES_BIONIC_POWER``` The item has no charges of its own, and runs off of the player's bionic power.
+- ```USE_PLAYER_ENERGY``` Item with `use_action` that `cast_spell` consumes the specified `base_energy_cost`.
```

Comment:
From the player's stats correct?  As an example if the item casts a spell that costs 100 mana, then it takes 100 mana from the player's mana total?   What if the player can't cast spells ? do they have mana then?  An example how to use this might be helpful.

Reply:
> From the player's stats correct? As an example if the item casts a spell that costs 100 mana, then it takes 100 mana from the player's mana total? 

Yeah, the item drains from the player's mana pool, in this case 200 from 1000 = 800.

> What if the player can't cast spells ? do they have mana then? An example how to use this might be helpful.

The spell itself doesn't need to be known by the player, the cast comes "from" the item itself.
Though, I was testing this and seems there's a problem. The player can still ```a```ctivate casts while on 0 mana. So, the flag works by draining mana, but doesn't prevent casting when the player is out of energy.

![5](https://user-images.githubusercontent.com/95231551/184785824-95dc12ca-a35d-4601-a794-5d65d53cbd6f.png)

Reply:
OK cool that's a separate issue but it's good to add documentation of the feature.